app/timetables/admin.py

```diff
@@ -1,3 +1,4 @@
 from django.contrib import admin
+from app.timetables.models import MealOption
```

Comment:
Also, please add an extra line before admin.site.register to follow our convention of two line spaces after imports